app/utils/GnosisProvider.js

```diff
@@ -28,7 +28,9 @@ class GnosisProvider extends Subprovider {
         return
 
       case 'eth_sendTransaction':
-        this.sendTransaction(payload, end)
+        (navigator.onLine)
```

Comment:
Is this really the preferred behavior? Wouldn't it make more sense to show the popup and allow the user to retry pulling the information necessary for sending the request? What is meta mask doing without internet?